examples/7guis/README.md

```diff
@@ -0,0 +1,62 @@
+# Slint's 7guis implementation
+
+[`7guis`](https://eugenkiss.github.io/7guis/) is a "GUI Programming Benchmark".
+However rather than benchmarking performance, it offers 7 GUI related tasks that aim to make UI Toolkits comparable.
+
+These 7 challenges have implementations for multiple frameworks already and the following are ours:
+
+## [Counter](https://eugenkiss.github.io/7guis/tasks#counter)
+Just a Button that increases a value in a text field.
+
+![Screenshot of the 7guis Counter](counter.png "Counter")
+
+[`.slint` code in web editor](https://slint-ui.com/editor/?load_url=https://raw.githubusercontent.com/slint-ui/slint/master/examples/7guis/counter.slint)
+
+## [Temperature Converter](https://eugenkiss.github.io/7guis/tasks/#temp)
+Converts Celsius to Fahrenheit and vice versa.
+
+![Screenshot of the 7guis Temperature Converter](tempconv.png "Temperature Converter")
```

Comment:
I wonder if it wouldn't look better without the explicit preferred-height and letting the line edit be the height of a normal line edit

Reply:
I wanted to do this. But I thought it caused weird resizing behavior with preview.
But actually resizing preview windows is broken either way...
At least on my system. The calculated minimum window size is wrong and it kind of glitches when you make it smaller.

Reply:
I opened it here: #1269